cblogger.go

```diff
@@ -110,28 +108,16 @@ func setRotateFileHook(loggerName string, logConfig *CBLOGCONFIG) {
 }
 
 func SetLevel(strLevel string) {
-	err := checkLevel(strLevel)
+
+	level, err := logrus.ParseLevel(strLevel)
 	if err != nil {
-		logrus.Errorf("Failed to set log level: %v", err)
+		thisLogger.logrus.Errorf("Failed to set log level: %v", strLevel)
+		thisLogger.logrus.Info("Set to Debug, the default logging level.")
```

Comment:
```suggestion
		thisLogger.logrus.Info("Not available log level: %v. Default log level will be used: debug", strLevel)
```
에러보다는 정보로만 제공하는 것이 어떨까요? ^^

Reply:
검토 감사드립니다.

제안해주신 `thisLogger.logrus.Info("Not available log level: %v. Default log level will be used: debug", strLevel)를 보니 **Warn레벨이 적절**할 것 같습니다.

말씀해 주신 것 처럼 debug level로 설정하였으니 에러로 보긴 어려울 것 같고요.

정보 제공으로 처리하기에는 모르고 지나칠 수 있기 때문에 error와 info의 중간인 warn이 적절 할 것 같다고 생각했습니다. 

제안해 주신 텍스트("Not available log level: %v. Default log level will be used: debug")도 warn에 아주 잘 어울린다고 생각합니다. ㅎㅎ

Reply:
`thisLogger.logrus.Warnf("Not available logging level: %v. Default logging level will be used: debug", strLevel)` 로 수정하여 Commit & Push 하겠습니다